### PR TITLE
Feature/issue details enhancement

### PR DESCRIPTION
**Pull request by moses-kabungo**
Changes in the PR includes
* Location/Address location is included in the issue details activity
* all the time labels are formatted using a common format
* Use coordinator layout to improve the scrolling behavior

### CODE REVIEW

File: app/src/main/java/com/github/codetanzania/adapter/IssueProgressTimelineAdapter.java [hunk 1]
```diff
@@ -72,7 +71,7 @@ public int getItemViewType(int position) {
 
         void bind(Comment comment) {
             mStatusContent.setText(comment.content);
-            mStatusTimestamp.setText(Util.formatDate(comment.timestamp, "MMM dd HH:mm"));
+            mStatusTimestamp.setText(LocalTimeUtils.formatDate(comment.timestamp, LocalTimeUtils.FMT_SHORT_DATE_TIME));
```
Comment (krtonga) [inline app/src/main/java/com/github/codetanzania/adapter/IssueProgressTimelineAdapter.java:74]: Better. I'm going to need to update the date format on the list fragment, so I can create one utility method.

File: app/src/main/java/com/github/codetanzania/ui/activity/IssueProgressActivity.java [hunk 2]
```diff
@@ -95,7 +98,7 @@ public void onDestroy() {
     }
 
     private void setupActionBar(ServiceRequest request) {
-        Toolbar toolbar = (Toolbar) findViewById(R.id.basic_toolbar_layout);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
```
Comment (krtonga) [inline app/src/main/java/com/github/codetanzania/ui/activity/IssueProgressActivity.java:101]: There is an issue with the Location Fragment - it doesn't show steps on first load. I have created a ticket in Tiaga (#89), but it can be a separate PR or incorporated in your next task.

File: app/src/main/java/com/github/codetanzania/util/LocalTimeUtils.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+public class LocalTimeUtils {
+
+    public static final String FMT_FULL_DATE_TIME  = "yyyy-MM-dd HH:mm:ss";
+    public static final String FMT_SHORT_DATE_TIME = "MMM dd HH:mm";
+
+    public static String formatDate(
+            @NonNull Date d, String pattern) {
+        SimpleDateFormat sdf = new SimpleDateFormat(pattern, Locale.US);
+        return sdf.format(d);
+    }
```
Comment (krtonga) [inline app/src/main/java/com/github/codetanzania/util/LocalTimeUtils.java:18]: I will add utility methods here when I update list view date, if that is ok with you.

File: app/src/main/res/layout/card_view_map.xml [hunk 4]
```diff
@@ -4,9 +4,10 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="200dp"
+    android:layout_height="350dp"
```
Comment (krtonga) [inline app/src/main/res/layout/card_view_map.xml:7]: If this is in two places, it probably should be a dimen.

File: app/src/main/res/values/styles.xml [hunk 5]
```diff
@@ -14,6 +14,21 @@
...
+        <item name="windowActionBarOverlay">true</item>
+    </style>
+
+    <style name="TransparentActionBar"
+        parent="Widget.AppCompat.Light.ActionBar.Solid">
+        <item name="android:background">#666666</item>
+        <!-- Support library compatibility -->
+        <item name="background">@color/colorHeavyGray</item>
+    </style>
+
```
Comment (krtonga) [inline app/src/main/res/values/styles.xml:31]: There's something still off with this styling, which one of us should probably work on. See  user story #90 here https://tree.taiga.io/project/krtonga-dawasco-app/us/90?milestone=145381

**Review by krtonga — APPROVED** [review]
So, some issues have come up with app bar, but I have separated them out into separate tasks. I will merge this, and we can fix app bar in the next.

File: app/src/main/java/com/github/codetanzania/ui/activity/IssueProgressActivity.java (hunk 2 above)
Comment (moses-kabungo) [reply app/src/main/java/com/github/codetanzania/ui/activity/IssueProgressActivity.java:101]: I think I may have a clue

File: app/src/main/res/values/styles.xml (hunk 5 above)
Comment (moses-kabungo) [reply app/src/main/res/values/styles.xml:31]: We should discuss first. Part of the reason I blv is that we are lacking design guidelines.
